src/contexts/AppContext.jsx

```diff
@@ -8,17 +8,20 @@ export const AppContext = createContext();
  * AppProvider
  */
 export function AppProvider(props) {
-  const { children, tourItems: tourItemsValue } = props;
+  const {
+    children,
+    tourItems: tourItemsValue,
+    navPages: navPagesValue,
+  } = props;
   const [tourTitle, setTourTitle] = useState("");
   const [tourDescription, setTourDescription] = useState("");
-  // Although we could use an Array using a Map allows us to
-  // use the id as the key and makes the value easier to access
+  const [navPages, setNavPages] = useState(navPagesValue || []);
+  const [activeNavPage, setActiveNavPage] = useState(0);
   const [tourItems, tourItemsDispatch] = useReducer(
     tourItemsReducer,
-    new Map(
-      tourItemsValue ? tourItemsValue.map((item) => [item.id, item]) : [],
-    ),
+    tourItemsValue || [],
   );
+  const navSearchButtonRef = useRef(null);
 
```

Comment:
Check if this can be moved out of AppContext

Reply:
Gonna leave this in `AppContext` as it gets set in `<Navigation/>` and read in `<TourItems/>`, but there isn't a direct hierarchy that could pass this down apart from the `AppContext`. Looking at this now it seems logical